### ISSUE TITLE
add default moduleclass for AI/ML software?

### ISSUE THREAD

**Issue by migueldiascosta** [easybuilders/easybuild#815]
currently, AI/ML software is categorized rather loosely: scikit-learn is "data", pytorch is "devel", tensorflow is "lib", keras is "math"...

there's nothing about AI/ML in the list of default moduleclasses, https://github.com/easybuilders/easybuild-framework/blob/454fe5e024b443b189f8e0f9f8727ed54c28ff56/easybuild/tools/config.py#L400-L424 (but it's not that they are not specific, there's even `quantum` :) )

should we do something about this?

**Comment by boegel** [comment]
Yeah, the time is right to add a new default module class and start using it. Do we want to go with `ai`, or `ai_ml`, etc.?

I guess `ai` is broad enough to also cover machine learning?

**Comment by migueldiascosta** [comment]
Another question is if we would reclassify existing easyconfigs or just new ones

**Comment by boegel** [comment]
I think re-classifying makes sense. PRs like that are trivial enough to merge without requiring test reports for all easyconfigs we touch (which can be brutally expensive for software like TensorFlow and PyTorch...)
